Misc/NEWS.d/next/Library/2018-02-21-11-59-21.bpo-32893.bLWZNi.rst

```diff
@@ -0,0 +1,2 @@
+ast.literal_eval() no longer accepts booleans as numbers when pass AST
```

Comment:
Is there a need for `when pass AST argument.` part? You can give argument as string (like `ast.literal_eval("True+6j")` and still get `ValueError`.

Reply:
You get ValueError in `ast.literal_eval("True+6j")`. This PR only changes the behavior of `ast.literal_eval()` when pass AST argument.

Reply:
Ah,  I see.